pdns/recursordist/rec-main.cc

```diff
@@ -2893,9 +2894,43 @@ int main(int argc, char** argv)
     g_slogout = g_slog->withName("out");
 
     ::arg().setSLog(startupLog);
+
+    if (::arg().mustDo("config")) {
+      string config = ::arg()["config"];
+      if (config == "default") {
+        cout << ::arg().configstring(false, true);
+      }
+      else if (config == "check") {
+        try {
+          if (!::arg().file(configname.c_str())) {
+            SLOG(g_log << Logger::Warning << "Unable to open configuration file '" << configname << "'" << endl,
+                 startupLog->error("No such file", "Unable to open configuration file", "config_file", Logging::Loggable(configname)));
+            exit(1);
+          }
+          ::arg().parse(argc, argv);
+          exit(0);
+        }
+        catch (const ArgException& argException) {
+          SLOG(g_log << Logger::Warning << "Unable to parse configuration file '" << configname << "': " << argException.reason << endl,
+               startupLog->error("Cannot parse configuration", "Unable to parse configuration file", "config_file", Logging::Loggable(configname), "reason", Logging::Loggable(argException.reason)));
+          exit(1);
+        }
+      }
+      else {
```

Comment:
This is for the `--confg=xyz` (where `xyz` may be empty) case. The man page change does not describe this.
While looking how auth describes this I see it also has `diff`, which would be really nice to have in rec as well 

So can you take the man pages lines from auth and see if `diff` can be added as well?

Reply:
Alright

Reply:
Is it better now?